backend/app/controllers/api/v1/admin/wage_chunk_reporting_tags_controller.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+class Api::V1::Admin::WageChunkReportingTagsController < ApplicationController
+    before_action :find_wage_chunk
+
+    # GET /wage_chunks/:wage_chunk_id/reporting_tags
+    def show
+        render_success @wage_chunk.reporting_tags
+    end
+
+    # POST /wage_chunks/:wage_chunk_id/reporting_tags
+    def create
+        start_transaction_and_rollback_on_exception do
+            reporting_tag =
+                ReportingTag.find_or_create_by(name: reporting_tag_name)
+            reporting_tag.save!
+
+            # A reporting tag can show up at most once in a reporting tags list,
+            # so check if it's already included
+            unless @wage_chunk.reporting_tag_ids.include?(reporting_tag.id)
+                @wage_chunk.reporting_tags << reporting_tag
+            end
+            @wage_chunk.save!
```

Comment:
Can we move this save action inside the unless clause since it is only updated inside?